apps/prairielearn/public/stylesheets/local.css

```
@@ -154,6 +154,25 @@ mjx-container {
   --bs-btn-disabled-border-color: #f8f9fa;
 }
 
+/**
+ * Similar to `btn-light`, but with a slightly darker background.
+ */
+.btn.btn-med-light {
+  --bs-btn-color: #292121;
+  --bs-btn-bg: #dbdcdc;
+  --bs-btn-border-color: #dbdcdc;
+  --bs-btn-hover-color: #292121;
+  --bs-btn-hover-bg: #ced0d0;
+  --bs-btn-focus-bg: #c3c3c3;
+  --bs-btn-focus-shadow-rgb: 195, 195, 195;
+  --bs-btn-active-color: #292121;
+  --bs-btn-active-bg: #c3c3c3;
+  --bs-btn-active-border-color: #c3c3c3;
+  --bs-btn-disabled-color: #292121;
+  --bs-btn-disabled-bg: #c3c3c3;
+  --bs-btn-disabled-border-color: #c3c3c3;
```

Just curious, how were all these values determined?

---

Ones that were explicitly stated before (in the lines being removed below) I was able to use but several of these I was not sure where to get a value from.  I used `#c3c3c3` from the "focus" background for active and disabled backgrounds as they looked similar to what was happening previously. Do we have actual values I can use for these?

---

> Do we have actual values I can use for these?

We didn't yet, but here's my proposal:

```suggestion
  --bs-btn-color: #000;
  --bs-btn-bg: #dee2e6;
  --bs-btn-border-color: #dee2e6;
  --bs-btn-hover-color: #000;
  --bs-btn-hover-bg: #bdc0c4;
  --bs-btn-hover-border-color: #b2b5b8;
  --bs-btn-focus-shadow-rgb: 189, 192, 196;
  --bs-btn-active-color: #000;
  --bs-btn-active-bg: #b2b5b8;
  --bs-btn-active-border-color: #a7aaad;
  --bs-btn-active-shadow: inset 0 3px 5px rgba(0, 0, 0, 0.125);
  --bs-btn-disabled-color: #000;
  --bs-btn-disabled-bg: #dee2e6;
  --bs-btn-disabled-border-color: #dee2e6;
```

These values are from rebuilding the Bootstrap CSS with the `light` base color redefined from `gray-100` to `gray-300` here:

https://github.com/twbs/bootstrap/blob/bafa9d25ed38b8742a3f52bc1401d46b71960153/scss/_variables.scss#L307

My theory is that this is better than doing it ourselves since this uses all Bootstrap's normal math to compute the different colors here.